CMakeLists.txt

```diff
@@ -60,12 +60,12 @@ set(PNET_MAX_CR                 2
 set(PNET_MAX_SLOTS            5
   CACHE STRING "Per API. Should be > 1 to allow at least one I/O module")
 set(PNET_MAX_SUBSLOTS         3
-  CACHE STRING "Per slot (DAP requires 2 + PNET_MAX_PORT)")
+  CACHE STRING "Per slot (DAP requires 2 + PNET_MAX_PHY_PORTS)")
```

Comment:
I prefer not to have `_MAX_` in the name, simply because if we have one port then it is not valid to run with `PNET_MAX_PHY_PORTS = 2`
Maybe
`PNET_PHY_PORTS` or `PNET_NUM_PHY_PORTS` or `PNET_NUMBER_OF_PORTS`

Reply:
Name is now changed